src/validations/constraints/unit-tests/iventory-item-or-component-has-virtual-FAIL.yaml

```
@@ -0,0 +1,9 @@
+test-case:
+  name: Negative Test for iventory-item-or-component-has-virtual
+  description: >-
+    This test case validates the behavior of constraint
+    iventory-item-or-component-has-virtual
+  content: ../content/ssp-iventory-item-or-component-has-virtual-INVALID.xml
+  expectations:
+    - constraint-id: iventory-item-or-component-has-virtual
+      result: fail
```

recommend renaming file to "inventory-item-or-component-has-virtual-FAIL.yaml", and making the following changes:

```suggestion
  name: Negative Test for inventory-item-or-component-has-virtual
  description: >-
    This test case validates the behavior of constraint
    inventory-item-or-component-has-virtual
  content: ../content/ssp-inventory-item-or-component-has-virtual-INVALID.xml
  expectations:
    - constraint-id: inventory-item-or-component-has-virtual
      result: fail
```

---

Resolved: [be0c369](https://github.com/GSA/fedramp-automation/pull/1055/commits/be0c36931135fabf91418bb5fdb71eb02dee8cc5)